content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -412,21 +436,14 @@ The SNI proxy will forward the traffic to port `443`.
       - "*.wikipedia.org"
       ports:
       - number: 443
-        name: tls
-        protocol: TLS
+        name: https
```

Comment:
Should be TLS, as in https://github.com/istio/istio.io/pull/8050/files#r479805216.